airflow/providers/amazon/aws/example_dags/example_glacier_to_gcs.py

```diff
@@ -46,6 +49,12 @@
     )
     # [END howto_sensor_glacier_job_operation]
 
+    # [START howto_operator_glacier_upload_data]
```

Comment:
This one would make more sense?

```suggestion
    # [START howto_operator_glacier_upload_archive]
```